internal/api/webhook_receivers.go

```diff
@@ -0,0 +1,30 @@
+package api
+
+// WebhookReceiverType is an enum representing the type of a webhook receiver.
+// It is used to identify the platform or service that the webhook receiver is
+// associated with, such as GitHub or Quay.
+type WebhookReceiverType string
+
+func (w WebhookReceiverType) String() string {
+	return string(w)
+}
```

Comment:
This isn't really needed. Variables of type `WebhookReceiverType` will already be treated like plain strings in most places and in the few places they aren't, typcasting is no harder than invoking `.String()`.